Formula/cuba.rb

```diff
@@ -0,0 +1,24 @@
+class Cuba < Formula
+  desc "Library for multidimensional numerical integration"
+  homepage "http://www.feynarts.de/cuba/"
+  url "http://www.feynarts.de/cuba/Cuba-4.2.tar.gz"
+  sha256 "6b75bb8146ae6fb7da8ebb72ab7502ecd73920efc3ff77a69a656db9530a5eef"
+
+  option "without-test", "Skip build-time tests (not recommended)"
+
+  def install
+    ENV.deparallelize
```

Comment:
Does parallel make fail? Has this bug been reported?

Reply:
Yes it fails. Frankly I don't know where the development is done, there's [something on Github](https://github.com/JohannesBuchner/cuba) but it doesn't seem to be official. Also, I don't think it's maintained anymore.

Reply:
Please report it to the author (http://wwwth.mpp.mpg.de/members/hahn/) by email, and say so in your comment